R/zzz.R

```diff
@@ -29,7 +29,7 @@ hintr_init_traduire <- function() {
   namespaces <- "translation"
   traduire::translator_register(resources = NULL,
                                 language = languages[[1]],
-                                default_namspace = namespaces[[1]],
+                                default_namespace = namespaces[[1]],
```

Comment:
oh dear. How was this not caught? does the line not actually do anything?

Reply:
I think the default in traduire is to use `"translation"` as the default namespace when not set, so this change doesn't actually make a difference